backend/geneaprove/views/queries.py

```diff
@@ -172,30 +184,35 @@ def get_ancestors(cls, person_id, max_depth=None, skip=0):
         assert isinstance(person_id, int)
 
         with django.db.connection.cursor() as cur:
-            initial = f"VALUES({person_id},0)"
+            # ??? group_concat doesn't exist in postgres
+            if 'postgresql' in database_in_use:
+                initial = f"VALUES({person_id}::bigint, 0::bigint)"
+                group_concat = "string_agg(parents.parent::text, ',') AS parents "
+            else:
+                initial = f"VALUES({person_id},0)"
+                group_concat = "group_concat(parents.parent) AS parents "
             md = f"AND ancestors.generation<={max_depth} " if max_depth else ""
             sk = f"WHERE ancestors.generation>{skip} " if skip else ""
             q = (
                 "WITH RECURSIVE parents(main_id, parent) AS (" +
                     cls._query_get_parents() +
-                "), ancestors(main_id,generation) as ("
+                "), ancestors(main_id,generation) AS ("
                     f"{initial} "
                     "UNION "
                     "SELECT parents.parent, ancestors.generation + 1 "
                     "FROM parents, ancestors "
                     "WHERE parents.main_id = ancestors.main_id "
                     f"{md}"
                 ") SELECT ancestors.main_id, ancestors.generation, "
-
-                # ??? group_concat doesn't exist in postgres
-                "group_concat(parents.parent) as parents "
-
+                f"{group_concat}"
                 "FROM ancestors LEFT JOIN parents "
                 "ON parents.main_id=ancestors.main_id "
                 f"{sk}"
                 "GROUP BY ancestors.main_id, ancestors.generation"
             )
+            logger.debug(f'get_ancestors() query: {q}')
```

Comment:
Let's remove that, since we can see all queries by enabling DEBUG in the settings

Reply:
Yep. I used that while testing and intended to remove it before I pushed. :}